bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/providers/cli/CliProvider.java

```diff
@@ -196,13 +193,13 @@ public boolean hasConfigs() {
         throw new ConfigException(CONFIG_CLI_TYPE_NOT_SUPPORTED, key.variable, effectiveType);
     }
 
-    public Optional<BMap<BString, Object>> getAsMapAndMark(Module module, VariableKey key) {
+    public Optional<Object> getAsMapAndMark(Module module, VariableKey key) {
```

Comment:
`module` is not used

Reply:
This is a overridden method from `ConfigProvider` 
https://github.com/ballerina-platform/ballerina-lang/blob/974b4d45dcc87dbd203a569248b0d1f59fc2eaae/bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/configurable/ConfigProvider.java#L60

The `@Override` will be added in latest commit.